remote/startup.sh

```diff
@@ -7,6 +7,7 @@ set -exuo pipefail
 
 if [ "$(whoami)" == "root" ] || [ "$(whoami)" == "techuser" ];
 then
+    : "${RMT_PASSWORD:?}"
```

Comment:
What happens if `openssl passwd -6 -salt $(openssl rand -base64 16) "${RMT_PASSWORD:?}"` fails? This would still continue without error. We need to properly fail if the child command fails.

Reply:
Instead, use:

```sh
salt=$(openssl rand -base64 16)
hash=$(openssl passwd -6 -salt ${salt} "${RMT_PASSWORD:?}")
echo ${line%%:*}:$hash:${line#*:*:} > /etc/shadow;
```